CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ ARCHITECTURAL:
 * Project switched to using Go modules, while `vendor` is left for backwards build compatibility only. It is worth having a
 look at [README.md](README.md) to understand how Go modules impact build and development.
 
+IMPROVEMENTS:
+
+* Insert Media now supports force ejecting with running VM
```

Comment:
"with running VM" -> "on a running VM"

Overall, appreciate the CHANGELOG addition, thanks.

Reply:
Fixed